jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/security/PropertyUserStore.java

```diff
@@ -147,37 +108,14 @@ private Path extractPackedFile(Resource configResource) throws IOException
         return extractedPath;
     }
 
-    /**
-     * Set the Config Path from a {@link File} reference
-     *
-     * @param configFile the config file
-     */
-    public void setConfigFile(File configFile)
-    {
-        if (configFile == null)
-            _configPath = null;
-        else
-            _configPath = configFile.toPath();
-    }
-
-    /**
-     * Set the Config Path
-     *
-     * @param configPath the config path
-     */
-    public void setConfigPath(Path configPath)
-    {
-        _configPath = configPath;
-    }
-
     /**
      * @return the resource associated with the configured properties file, creating it if necessary
+     * @deprecated
      */
+    @Deprecated(forRemoval = true)
     public Resource getConfigResource()
```

Comment:
Can't you just remove this method right away?

Reply:
This is for a future PR, as it started to get quite involved.